icons/house-manor.svg

```diff
@@ -0,0 +1,21 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M9 6V2H5v4" />
+  <path d="M19 6V2h-4v4" />
+  <rect width="20" height="16" x="2" y="6" rx="2" />
+  <path d="M2 12h4" />
+  <path d="M6 22V12l5.5-6" />
+  <path d="m12.5 6 5.5 6v10" />
+  <path d="M18 12h4" />
+  <path d="M12 11h.01" />
+  <path d="M10 22v-5a2 2 0 1 1 4 0v5" />
```

Comment:
The roof does not scale particularly well: 
![image](https://github.com/lucide-icons/lucide/assets/17746067/3d18f596-e432-4752-a90e-00db2c579b61)